new-dti-website/components/bottom.tsx

```diff
@@ -108,9 +108,9 @@ const Bottom: React.FC = () => (
         <div className="flex flex-col gap-y-3 md:gap-y-4">
           <div className="flex flex-col gap-y-1 md:gap-y-2">
             <p className="text-neutral-200 text-xs font-bold uppercase lg:text-lg">outreach</p>
-            <h5 className="text-lg font-semibold md:text-xl lg:text-[32px]">
+            <h3 className="text-lg font-semibold md:text-xl lg:text-[32px]">
```

Comment:
```suggestion
            <h2 className="text-lg font-semibold md:text-xl lg:text-[32px]">
```

same comment as this?